ramalama

```diff
@@ -351,11 +351,20 @@ def main(args):
     ramalama_store = get_ramalama_store()
 
     try:
+        dryrun = False
+        if args[0] in ("--dryrun", "-d"):
```

Comment:
nit: I would not loose the -d short option, this will not be done enough to warrant, and at some point in the future we might want to use -d for a more common option.